recipe/meta.yaml

```diff
@@ -44,3 +50,4 @@ extra:
   recipe-maintainers:
     - sandorkertesz
     - iainrussell
+    - xylar
```

Comment:
I'm happy to help maintain if you want.  If no, no problem, I will remove myself.